src/chessboard/types/index.ts

```diff
@@ -165,10 +167,11 @@ export type ChessboardProps = {
    */
   clearPremovesOnRightClick?: boolean;
   /**
-   * Array of custom arrows to draw on the board. Each arrow within the array must be an array of length 2 with strings denoting the from and to square to draw the arrow e.g. [ ['a3', 'a5'], ['g1', 'f3'] ].
+   * Array of custom arrows to draw on the board. Each arrow within the array must be an array of length 2 with `Square` denoting the from and to square to draw the arrow e.g. [ ['a3', 'a5'], ['g1', 'f3', 'red'] ].
```

Comment:
please update as this is no longer accurate, each array doesn't have to be of length 2 and Square is no longer used for typing